src/Symfony/Component/Intl/Collator/Collator.php

```diff
@@ -90,7 +90,7 @@ public function __construct(?string $locale)
      *
      * @throws MethodArgumentValueNotImplementedException When $locale different than "en" or null is passed
      */
-    public static function create($locale)
+    public static function create(string $locale)
```

Comment:
The constructor allows `$locale` to be `null`. Also, the docblock tells me that `null` is a valid value.